makefile

```diff
@@ -103,7 +103,7 @@ LDFLAGS=
 
 CFLAGS+=-fdata-sections -ffunction-sections
 LDFLAGS+=-Wl,--gc-sections -Wl,--print-gc-sections -Wl,--no-print-gc-sections
-LDFLAGS_USB=-lusb
+LDFLAGS_USB=`pkg-config libusb-1.0 --libs`
```

Comment:
Please give a proper explanation in commit message as to why we are disabling usb scan.

Reply:
Well, libusb-1.0 has the nice function libusb_open_device_with_vid_pid()
which does everything needed to open that particular device, so I do need to scan the bus manually, it would be only interesting for debugging purposes. And lsusb -v gives you also the same output as you would get by bus scanning. So I was just too lazy to translate that to libusb-1.0 api and did it the quick way. If still want the bus scan, I can of course also convert that part.

Reply:
it will be nice to add to commit message with the info